Base_of_credit_card_vendors.txt

```diff
@@ -0,0 +1,2 @@
+Visa 13/16 4 
```

Comment:
These types of properties/data should be in one central place for all instances. It cannot be a separate file in microservice instances, because there will be a problem with ha. Adding a new vendor would have to be done in each instance.